### DISCUSSION TITLE
Question About Leaderboard

### DISCUSSION THREAD

**Discussion by wang975679801** [PaddlePaddle/PaddleSpatial#133] [BaiduKDDCup2022] [answered]
I wonder why my ranking improved? I didn't make any submission.

**Comment by ramber1836** [comment, marked as answer]
This is because the high-ranked submissions will be selected and evaluated from time to time to ensure the fairness of the results.

**Reply by wang975679801** [reply]
Does it mean when the high-ranked submission is evaluated, the leaderboard will display their previous results.

**Reply by AlvinAi96** [reply]
Same question as above, I just found out my highest submission disappeared from the LB. Could you explain the above question?

**Reply by ramber1836** [reply]
> Does it mean when the high-ranked submission is evaluated, the leaderboard will display their previous results.

Yes. The previous results would disappear for a while when the submission is being re-evaluated. 

> Same question as above, I just found out my highest submission disappeared from the LB. Could you explain the above question?

When your previous submission is under evaluation, the score of the previous evaluation disappears and will be updated after the evaluation.

**Reply by AlvinAi96** [reply]
If my previous submission is under your evaluation and then I submit another code. 
Whether my latest submission could be successfully evaluated during your re-evaluation?

**Reply by ramber1836** [reply]
> If my previous submission is under your evaluation and then I submit another code. Whether my latest submission could be successfully evaluated during your re-evaluation?

Yes. The evaluation is carried out case by case.
